src/contexts/AuthContext.jsx

```diff
@@ -36,6 +37,10 @@ export function AuthProvider({ children }) {
   const logOut = () => {
     return signOut(auth);
   };
+  // Reset Password
```

Comment:
This comment is a bit redundant since the function describes what it's doing perfectly
```suggestion
```